gh-pages/src/index.md

```diff
@@ -12,6 +12,8 @@ Baseline features are available across popular browsers. Baseline has two stages
 - **Newly available**: The feature works across the latest devices and browser versions. The feature might not work in older devices or browsers. Indicated with a blue icon.
 - **Widely available**: The feature is well established and works across many devices and browser versions. It’s been available across browsers for at least 2½ years (30 months). Indicated with a green icon.
 
+Prior to being newly available, a feature has **Limited Availability** when it's not yet available across all browsers.
```

Comment:
nit: We try to use sentence case, so I guess this should be lower case here:

```suggestion
Prior to being newly available, a feature has **limited availability** when it's not yet available across all browsers.
```

This would also be OK, to match the typical widgets:

```suggestion
Prior to being newly available, a feature has **Limited availability** when it's not yet available across all browsers.
```